.gitignore

```diff
@@ -11,3 +11,5 @@ sqlnet.log
 Gemfile.lock
 *.zip
 .vagrant
+.idea
+spec/support/custom_config.rb
```

Comment:
Not hard, just need to be more careful and change settings in my IDE.

*sent from mobile
2 cze 2015 18:57 "Yavor Nikolov" <notifications@github.com> napisał(a):

> In .gitignore
> <https://github.com/rsim/ruby-plsql/pull/83#discussion_r31551142>:
>
> > @@ -11,3 +11,5 @@ sqlnet.log
> >  Gemfile.lock
> >  *.zip
> >  .vagrant
> > +.idea
> > +spec/support/custom_config.rb
>
> Is it possible to fix that "No new line at end of file" warning? (If it's
> hard to do on Windows - let me know, I can push a fix for that).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rsim/ruby-plsql/pull/83/files#r31551142>.
>